spec/functionalitySpec.js

```diff
@@ -0,0 +1,6 @@
+describe("", function(){
+  var skribble = require("../js/index.js");
+  it("should return a div with the player name",function(){
+
```

Comment:
Are you adding tests into here?

Reply:
yes, I wanted to ask you how to add them for jquery, as i'm thinking of making a Game object with methods that return the various aspects of the game